jobsdb/jobsdb.go

```diff
@@ -1210,6 +1209,17 @@ func (jd *HandleT) getTableSize(jobTable string) int64 {
 }
 
 func (jd *HandleT) checkIfFullDS(ds dataSetT) bool {
+	var minJobCreatedAt time.Time
+	sqlStatement := fmt.Sprintf(`SELECT created_at FROM "%s" ORDER BY "job_id" ASC LIMIT 1`, ds.JobTable)
```

Comment:
Can't we use the same query to get the min id? Probably add a helper function to get min and max?

`sqlStatement = fmt.Sprintf(`SELECT MIN(created_at), MAX(created_at) from "%s"`, ds.JobTable)`

Reply:
done